src/docs/cookbook-payments/removing-stripe/index.md

```diff
@@ -0,0 +1,332 @@
+---
+title: How to remove Stripe and payments
+slug: removing-stripe-and-payments
+updated: 2020-07-06
+category: cookbook-payments
+ingress:
+  In some cases, you might want to remove Stripe integrations or even all the
+  payments from FTW. This article gives you a starting point for these
+  customizations.
+published: true
+---
+
+# Removing Stripe
+
+In this guide, we will go through the minimum changes needed to make the booking
+flow work without Stripe. Most likely you need to customize some parts (e.g.
```

Comment:
I think this should include a bit more about what to expect. This guide is very rough by nature - clients clearly need to go through lots of (unguided) changes on several files. So, this paragraph could prepare devs & admins that going through these changes is going to be a relatively big customization project.


 FTW templates are made for a specific transaction process.
- Payments are a core feature that we have tried to cover with these provided templates.

```suggestion
flow work without Stripe. This is not an easy customization project. FTW templates are made for a specific transaction process and payments are a core feature: its ubiquitous nature means that you need to touch quite many components and files. Most likely you need to customize some parts (e.g.
```